client/my-sites/site-settings/header-code/index.jsx

```diff
@@ -0,0 +1,163 @@
+import { FEATURE_INSTALL_PLUGINS, getPlan, PLAN_BUSINESS } from '@automattic/calypso-products';
+import { Button, Card, Gridicon } from '@automattic/components';
+import classNames from 'classnames';
+import { useTranslate } from 'i18n-calypso';
+import { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import illustration404 from 'calypso/assets/images/illustrations/illustration-404.svg';
+import UpsellNudge from 'calypso/blocks/upsell-nudge';
+import DocumentHead from 'calypso/components/data/document-head';
+import QuerySiteSettings from 'calypso/components/data/query-site-settings';
+import EmptyContent from 'calypso/components/empty-content';
+import FormTextarea from 'calypso/components/forms/form-textarea';
+import HeaderCake from 'calypso/components/header-cake';
+import InlineSupportLink from 'calypso/components/inline-support-link';
+import Main from 'calypso/components/main';
+import NavigationHeader from 'calypso/components/navigation-header';
+import { successNotice, errorNotice } from 'calypso/state/notices/actions';
+import { canCurrentUser } from 'calypso/state/selectors/can-current-user';
+import getSiteSetting from 'calypso/state/selectors/get-site-setting';
+import siteHasFeature from 'calypso/state/selectors/site-has-feature';
+import { saveSiteSettings } from 'calypso/state/site-settings/actions';
+import {
+	isRequestingSiteSettings,
+	isSavingSiteSettings,
+	isSiteSettingsSaveSuccessful,
+	getSiteSettingsSaveError,
+} from 'calypso/state/site-settings/selectors';
+import { isJetpackSite } from 'calypso/state/sites/selectors';
+import { getSelectedSiteId, getSelectedSiteSlug } from 'calypso/state/ui/selectors';
+import './style.scss';
+
+const HeaderCodeSettings = () => {
+	const translate = useTranslate();
+	const dispatch = useDispatch();
+
+	const [ editedCode, setEditedCode ] = useState( '' );
+
+	const siteId = useSelector( ( state ) => getSelectedSiteId( state ) );
+	const hasPlanFeature = useSelector( ( state ) =>
+		siteHasFeature( state, siteId, FEATURE_INSTALL_PLUGINS )
```

Comment:
What does plugin installation have to do as a feature with inserting code to your `<head />`? Is that intentional?

Reply:
I only used it since sites with that features are the ones which can insert code, but I've removed it now since the nudge is gone.